mne/event.py

```diff
@@ -885,3 +886,349 @@ def concatenate_events(events, first_samps, last_samps):
+                              eog=250e-6 # V (EOG channels)
+                              )
+
+            If True, use the rejection limits defined in DACQ.
+            Note that all DACQ rejection criteria are not supported by mne.
+        flat : True | dict | None
+            Rejection parameters based on flatness of signal.
+            Valid keys are 'grad' | 'mag' | 'eeg' | 'eog' | 'ecg', and values
+            are floats that set the minimum acceptable peak-to-peak amplitude.
+            If True, use the flatness rejection limits defined in DACQ.
+            If None, no rejection is done.
+        baseline : None or tuple of length 2 (default (None, 0))
+            The time interval to apply baseline correction.
+            If None do not apply it. If baseline is (a, b)
+            the interval is between "a (s)" and "b (s)".
+            If a is None the beginning of the data is used
+            and if b is None then b is set to the end of the interval.
+            If baseline is equal to (None, None) all the time
+            interval is used.
+            The baseline (a, b) includes both endpoints, i.e. all
+            timepoints t such that a <= t <= b.
+        detrend : int | None
+            If 0 or 1, the data channels (MEG and EEG) will be detrended when
+            loaded. 0 is a constant (DC) detrend, 1 is a linear detrend. None
+            is no detrending. Note that detrending is performed before baseline
+            correction. If no DC offset is preferred (zeroth order detrending),
+            either turn off baseline correction, as this may introduce a DC
+            shift, or set baseline correction to use the entire time interval
+            (will yield equivalent results but be slower).
+        stim_channel : None | string | list of string
+            Name of the stim channel or all the stim channels
+            affected by the trigger. If None, the config variables
+            'MNE_STIM_CHANNEL', 'MNE_STIM_CHANNEL_1', 'MNE_STIM_CHANNEL_2',
+            etc. are read. If these are not found, it will fall back to
+            'STI 014' if present, then fall back to the first channel of type
+            'stim', if present.
+        mask : int | None
+            The value of the digital mask to apply to the stim channel values.
+            If None (default), no masking is performed.
+        uint_cast : bool
+            If True (default False), do a cast to ``uint16`` on the channel
+            data. This can be used to fix a bug with STI101 and STI014 in
+            Neuromag acquisition setups that use channel STI016 (channel 16
+            turns data into e.g. -32768), similar to ``mne_fix_stim14 --32``
+            in MNE-C.
+        mask_type: 'and' | 'not_and'
+            The type of operation between the mask and the trigger.
+            Choose 'and' for MNE-C masking behavior.
+        """
+
+        from .epochs import Epochs
```

Comment:
ok, it makes sense. The alternative would be to put all this into a separate file I guess. I don't have any opinion on this kind of issues.